src/config/section/storage.js

```diff
@@ -101,6 +101,12 @@ export default {
           label: 'label.action.attach.disk',
           message: 'message.confirm.attach.disk',
           args: ['virtualmachineid'],
+          mapping: {
+            virtualmachineid: {
+              api: 'listVirtualMachines',
+              params: (record) => { return { zoneid: record.zoneid, domainid: record.domainid, account: record.account, hypervisor: record.hypervisor } }
```

Comment:
@Pearl1594 can you check if this causes regression in case of a project; i.e. select a project and test the case and see if that works?

Reply:
This will require some additional checks.